MultiSlider.js

```diff
@@ -336,6 +344,11 @@ export default class MultiSlider extends React.Component {
       ? selectedStyle || styles.selectedTrack
       : unselectedStyle;
     const Marker = this.props.customMarker;
+
+      const MarkerLeft = this.props.customMarkerLeft;
+      const MarkerRight = this.props.customMarkerRight;
```

Comment:
wrong indentation here

Reply:
Updated in last commit:
https://github.com/sadeghipour/react-native-multi-slider/blob/master/MultiSlider.js#L348